docs/design/tcp-and-http-load-balancing.adoc

```diff
@@ -369,67 +375,68 @@ kubernetes.io/ingress.class: edgelb
 ----
 
 Like in the case of `Service` resources of type `LoadBalancer`, `Ingress` resources are exposed _externally_ by default.
-In order for a given `Ingress` resource to be exposed internally it *MUST* be explicitly annotated with
+In order for a given `Ingress` resource to be exposed internally it *MUST*specify the following configuration object:
 
 [source,text]
 ----
-kubernetes.dcos.io/edgelb-pool-role: "<edgelb-pool-role>"
+kubernetes.dcos.io/dklb-config: |
+role: "<edgelb-pool-role>"
```

Comment:
Sometimes strings are double-quoted, some other times they're not. Please, normalize.

Reply:
YAML has complex requirements when it comes to the need for double-quotes... My usual rule is:

> Use quotes if your value includes special characters, (e.g. `:`, `{`, `}`, `[`, `]`, `,`, `&`, `*`, `#`, `?`, `|`, `-`, `<`, `>`, `=`, `!`, `%`, `@`, `\`).

But I understand this is confusing, so, as mentioned above, I'll use double-quotes to explicitly indicate a string everywhere where the configuration object appears.

Reply:
Interesting because in many examples here you don't seem to abide to this rule, eg
```
kubernetes.dcos.io/dklb-config: |  # NOTE: The "|" character is mandatory.
  name: dklb-pool-0
(...)
```